.github/workflows/coverage.yml

```diff
@@ -35,7 +33,12 @@ jobs:
       - name: 'Run coverage'
         if: steps.check_context.outputs.run_coverage == 'true'
         continue-on-error: true
-        run: "PATH=/opt/llvm/bin:${PATH} ./envoy/test/run_envoy_bazel_coverage.sh //test/common/..."
+        run: |
+          echo "build --remote_header=\"Authorization=Bearer ${{ secrets.GITHUB_TOKEN }}\"" > ~/.bazelrc
```

Comment:
Doesn't this overwrite the checked-in `.bazelrc`? Shouldn't it use `>>`?

Reply:
From what I could tell the home bazelrc in the GH runner is always empty and I wanted to avoid adding flags on top of existing ones if the bazelrc exists. I can change it if you think it's better but I'm ok with it as it is.

Reply:
Ah, didn't read that correctly.